Docker/start-testnet.sh

```diff
@@ -73,7 +73,9 @@ docker run --rm --network cyberway_cyberway-net -ti $GOLOS_IMAGE /bin/bash -c 'r
 call_hook "nodeos-started"
 
 echo "=== Deploy cyberway & golos contracts"
-docker run --rm --network cyberway_cyberway-net -ti $GOLOS_IMAGE /opt/golos.contracts/scripts/boot-sequence.py
+if [[ -n "$RUN_BOOT_SEQUENCE" ]]; then
+    docker run --rm --network cyberway_cyberway-net -ti $GOLOS_IMAGE /opt/golos.contracts/scripts/boot-sequence.py
+fi
 call_hook "contracts-loaded"
```

Comment:
Should we move that `call_hook` inside `if` block too?

Reply:
Echo - Ok.
call_hook - more difficult question. When testnet start with genesis, contracts already loaded. So hook `contracts-loaded` actually. But as I know, it's never used now, so I move it inside `if` too.